patterns/behavioural/command.md

```diff
@@ -0,0 +1,226 @@
+# Command
+
+## Description
+
+Suppose we have a sequence of actions or transactions
+encapsulated as objects.
+We want these actions or commands to be
+executed or invoked in some order later at different time.
+These commands may also be triggered as a result
+of some event. For example when a user pushes a button
```

Comment:
```suggestion
of some event. For example, when a user pushes a button,
```

Unsure about the second one, maybe an English native can check for the Oxford comma thingy